src/AutoRest.CSharp/Common/Generation/Writers/FormattableStringHelpers.cs

```diff
@@ -133,8 +127,33 @@ public static FormattableString GetConversionFormattable(this Parameter paramete
             return $"{parameter.Name:I}{conversionMethod}";
         }
 
+        private static FormattableString GetConversionFromFrameworkToRequestContent(this Parameter parameter, string? contentType)
+        {
+            if (TypeFactory.IsReadWriteDictionary(parameter.Type))
```

Comment:
Does array (such as string[] )can be text? current we convert all array/list to Json.
Same consider about the enum.

Reply:
All of these are out of the scope of this PR.